pkg/routeagent_driver/main.go

```diff
@@ -60,9 +55,17 @@ func main() {
 		klog.Errorf("Error while annotating the node: %s", err.Error())
 	}
 
-	registry := event.NewRegistry("routeagent_driver", os.Getenv("NETWORK_PLUGIN"))
-	if err := registry.AddHandlers(logger.NewHandler(),
-		kp_iptables.NewSyncHandler(env.ClusterCidr, env.ServiceCidr, smClientset)); err != nil {
+	np := os.Getenv("SUBMARINER_NETWORKPLUGIN")
```

Comment:
Not specific to this review, but we should put these constants in a common file which can be reused in operator to avoid mismatch.

Reply:
yup, may be in admiral. Sounds like a good idea.